src/Illuminate/Http/Middleware/PassThroughTrait.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Illuminate\Http\Middleware;
+
+trait PassThroughTrait
+{
+    /**
```

Comment:
http://php.net/manual/en/language.oop5.traits.php

> If a trait defines a property then a class can not define a property with the same name, otherwise an error is issued. It is an E_STRICT if the class definition is compatible (same visibility and initial value) or fatal error otherwise.

Example #12 Conflict Resolution
```php
<?php
trait PropertiesTrait {
    public $same = true;
    public $different = false;
}

class PropertiesExample {
    use PropertiesTrait;
    public $same = true; // Strict Standards
    public $different = true; // Fatal error
}
?>
```